File: quests/airplane/Keeper_of_Souls.pl
# Epic NPC -- Keeper_of_Souls

sub EVENT_SPAWN {
  quest::settimer("summit",1800);
}

sub EVENT_TIMER {
  if ($timer eq "summit") {
    quest::stoptimer("summit");
    quest::depop();
  }
}

sub EVENT_DEATH_COMPLETE {
  $sirran= undef;
  quest::setglobal("sirran",4,3,"M20");
  quest::spawn2(71058,0,0,-543,767,174,64);
} 

#END of FILE  Quest by: Solid11  Zone:airplane  ID:71075 -- Keeper_of_Souls

File: airplane/Keeper_of_Souls.lua
-- Epic NPC -- Keeper_of_Souls

function event_spawn(e)
	eq.set_timer("summit",1800000);
end

function event_timer(e)
	if(e.timer == "summit") then
		eq.stop_timer("summit");
		eq.depop();
	end
end

function event_death_complete(e)
	sirran= nil;
	eq.set_global("sirran","4",3,"M20");
	eq.spawn2(71058,0,0,-543,767,174,128); -- NPC: Sirran_the_Lunatic
end

--Quest by: Solid11
-------------------------------------------------------------------------------------------------
-- Converted to .lua using MATLAB converter written by Stryd
-- Find/replace data for .pl --> .lua conversions provided by Speedz, Stryd, Sorvani and Robregen
-------------------------------------------------------------------------------------------------
